demo/rails/Gemfile

```
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
 gem 'dotenv-rails', groups: [:development, :test]
-gem 'rails', '4.2.5'
+gem 'rails', '~> 4.2.11.1'
```

any v4 should do, so this is better/less locked down:
```suggestion
gem 'rails', '~> 4.2'
```